src/components/shared/InlineMessage.js

```diff
@@ -1,54 +1,41 @@
-import { Box, Paragraph, Text } from 'grommet'
-import styled, { css } from 'styled-components'
-import InfoIcon from '../../images/info.svg'
-import SuccessIcon from '../../images/success.svg'
-import ErrorIcon from '../../images/warning.svg'
+import { Box, Paragraph } from 'grommet'
+import { Icon } from 'components/shared/Icon'
 
 // status: error, info, success
-const SVGs = {
-  InfoIcon,
-  SuccessIcon,
-  ErrorIcon
-}
-
-const P = styled(Paragraph)`
-  ${({ theme }) => css`
-    color: ${(props) => theme.global.colors[props.color]};
-  `}
-`
 
 export const InlineMessage = ({
+  color = 'info',
+  height = '24px',
   label = '',
   labelOnly = false,
-  color = 'info',
+
   ...props
 }) => {
-  const SVGIcon =
-    SVGs[`${color.substring(0, 1).toUpperCase()}${color.substring(1)}Icon`]
-  const errorIconColor = 'coral-shade-20'
+  const SVG =
```

Comment:
David, Thank you for the suggestions. 

For the whitelist `...props`, it was added to utilize the Grommet's built-in default props (instead of introducing extra markups (e.g., `div`) in JSX).

The most recent update to the [`InlineMessage`](https://github.com/AlexsLemonade/refinebio-web/blob/33202d5289c0fa66520ed7e14d28ca0fadf92c6a/src/components/shared/InlineMessage.js) component is included PR [36 - Implement Samples Table](https://github.com/AlexsLemonade/refinebio-web/pull/41). For this reason, I'll address any adjustments as needed in that PR review (left a comment about this in the PR), thank you!